packages/cms/pages/auth/register/register.tsx

```diff
@@ -31,36 +49,58 @@ export const Register: React.FunctionComponent = () => {
 
       <EuiSpacer size="xl" />
 
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow label="First name">
-            <EuiFieldText autoFocus />
-          </EuiFormRow>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow label="Last name">
-            <EuiFieldText />
-          </EuiFormRow>
-        </EuiFlexItem>
-      </EuiFlexGroup>
+      <form method="post" onSubmit={submitRegistration}>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFormRow label="First name">
+              <EuiFieldText
+                autoFocus
+                onChange={e => {
+                  setUser({ ...user, firstname: e.target.value })
+                }}
+              />
+            </EuiFormRow>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFormRow label="Last name">
+              <EuiFieldText
+                onChange={e => {
+                  setUser({ ...user, lastname: e.target.value })
+                }}
+              />
+            </EuiFormRow>
+          </EuiFlexItem>
+        </EuiFlexGroup>
 
-      <EuiSpacer size="l" />
+        <EuiSpacer size="l" />
 
-      <EuiFormRow label="Email">
-        <EuiFieldText fullWidth />
-      </EuiFormRow>
+        <EuiFormRow label="Email">
+          <EuiFieldText
+            fullWidth
+            onChange={e => {
+              setUser({ ...user, email: e.target.value })
+            }}
+          />
+        </EuiFormRow>
 
-      <EuiSpacer size="l" />
+        <EuiSpacer size="l" />
 
-      <EuiFormRow label="Password">
-        <EuiFieldPassword type="dual" fullWidth />
-      </EuiFormRow>
+        <EuiFormRow label="Password">
+          <EuiFieldPassword
+            type="dual"
+            fullWidth
+            onChange={e => {
+              setUser({ ...user, password: e.target.value })
+            }}
+          />
+        </EuiFormRow>
 
-      <EuiSpacer size="l" />
+        <EuiSpacer size="l" />
```

Comment:
These spaces seem like reformatting the code. I recommend running `yarn prettier` to keep the formatting.

Reply:
Okay, thanks